### DISCUSSION TITLE
Default dust_emis_fact was changed 10 times higher?

### DISCUSSION THREAD

**Discussion by lxu16** [E3SM-Project/E3SM#6124] [E3SM model help]
The constant dust_emis_fact is now 12.8 in the default namelist of EAM. That is nearly 10 times higher than 1.5 previously used. I could not find any source/reference about this big change. Does anybody direct me the source? BTW, I checked my dust burden and dust AOD which is very small. That is not correct, not sure if it is related to this dust_emis_fact.

**Comment by rljacob** [comment]
@yfenganl can you answer this?

**Comment by mahf708** [comment]
A new dust scheme was used in v3; and two new settings are recommended for v3 EAM:  dust_emis_scheme = 2 and dust_emis_fact = 13.8. You can find more information about the new dust scheme in our docs here: https://docs.e3sm.org/E3SM/EAM/tech-guide/dust/.
